pkg/wallet/api.go

```diff
@@ -56,6 +47,9 @@ type Crypto interface {
 	//
 	//		in case of error, the index will be -1
 	FindVerKey(candidateKeys []string) (int, error)
+
+	// GetEncryptionKey will return the public encryption key corresponding to the public verKey argument
+	GetEncryptionKey(verKey []byte) ([]byte, error)
```

Comment:
I found this method name a bit confusing.

Reply:
it's to get the encryption key corresponding to the signing key

Reply:
maybe `GetSubKey(keyIdentifier, subKeyType)`?

Reply:
this will be refactored as part of #596